addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -21,6 +21,10 @@ _thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIn
 // it's already dead, who cares?
 if (damage _vehicle >= 1) exitWith {};
 
+// Check for players and exit if none found and the disable for AI setting is true
+private _playerCrew = (fullCrew [_vehicle, "", false]) findIf {isPlayer (_x #0)};
+if (_vehicle getVariable [QGVAR(disableForAI), GVAR(disableForAI)] && _playerCrew isEqualTo -1) exitWith {};
```

Comment:
`_playerCrew` is such an arbitrary variable name. It is not clear what the content of the variable is. If taken literally it is misleading, because the name would imply something like an array of all crew that happen to be players.

```sqf
private _hasPlayerInCrew = fullCrew [_vehicle, "", false] findIf {isPlayer (_x select 0)} != -1;
if (_vehicle getVariable [QGVAR(disableForAI), GVAR(disableForAI)] && !_hasPlayerInCrew) exitWith {};
```